Libplanet.RocksDBStore/RocksDBKeyValueStore.cs

```diff
@@ -27,34 +29,52 @@ public RocksDBKeyValueStore(string path)
         }
 
         /// <inheritdoc/>
-        public byte[] Get(byte[] key) => _keyValueDb.Get(key) ?? throw new KeyNotFoundException(
-            $"There were no elements that correspond to the key (hex: {ByteUtil.Hex(key)}).");
+        public byte[] Get(in KeyBytes key) => _keyValueDb.Get(key.ToByteArray())
+            ?? throw new KeyNotFoundException($"No such key: ${key}.");
+
+        /// <inheritdoc cref="IKeyValueStore.Get(IEnumerable{KeyBytes})"/>
+        public IReadOnlyDictionary<KeyBytes, byte[]> Get(IEnumerable<KeyBytes> keys)
+        {
+            byte[][] keyArray = keys.Select(k => k.ToByteArray()).ToArray();
+            KeyValuePair<byte[], byte[]?>[] pairs = _keyValueDb.MultiGet(keyArray);
+            var dictBuilder = ImmutableDictionary.CreateBuilder<KeyBytes, byte[]>();
+            foreach (KeyValuePair<byte[], byte[]?> pair in pairs)
+            {
+                if (pair.Value is { } b)
+                {
+                    dictBuilder[new KeyBytes(pair.Key)] = b;
+                }
+            }
+
+            return dictBuilder.ToImmutable();
+        }
 
         /// <inheritdoc/>
-        public void Set(byte[] key, byte[] value)
+        public void Set(in KeyBytes key, byte[] value)
         {
-            _keyValueDb.Put(key, value);
+            _keyValueDb.Put(key.ToByteArray(), value);
         }
 
         /// <inheritdoc/>
-        public void Set(IDictionary<byte[], byte[]> values)
+        public void Set(IDictionary<KeyBytes, byte[]> values)
         {
             using var writeBatch = new WriteBatch();
 
-            foreach (KeyValuePair<byte[], byte[]> kv in values)
+            foreach (KeyValuePair<KeyBytes, byte[]> kv in values)
             {
-                writeBatch.Put(kv.Key, kv.Value);
+                writeBatch.Put(kv.Key.ToByteArray(), kv.Value);
             }
 
             _keyValueDb.Write(writeBatch);
         }
 
         /// <inheritdoc/>
-        public void Delete(byte[] key)
+        public void Delete(in KeyBytes key)
```

Comment:
Just a thought. Might be nice to have multi-key-delete just like `Get` and `Set`? 😶

Reply:
Good idea.  I'm going to give it a try!

Reply:
Added `IKeyValueStore.Delete(IEnumerable<KeyBytes>)` method: 43526ca3b72fe72933aaf660e628b5377e3fc211.